src/main/java/ee/tuleva/onboarding/auth/jwt/JwtTokenUtil.java

```diff
@@ -35,6 +40,11 @@ public JwtTokenUtil(
     final var keystore = KeyStore.getInstance("PKCS12");
     keystore.load(keystoreResource.getInputStream(), keystorePassword);
     this.signingKey = keystore.getKey("jwt", keystorePassword);
+    this.jwtParser =
+        Jwts.parserBuilder()
+            .setSigningKey(signingKey)
+            .setClock(() -> Date.from(clock.instant()))
+            .build();
```

Comment:
Should this be a bean defined in a @Configuration class that we can inject instead?